doc/users/prev_whats_new/whats_new_1.0.rst

```diff
@@ -69,7 +69,7 @@ figures since the last call.  Eric has done a lot of testing on the
 user interface toolkits and versions and platforms he has access to,
 but it is not possible to test them all, so please report problems to
 the `mailing list
-<http://mail.python.org/mailman/listinfo/matplotlib users>`__
+<http://mail.python.org/mailman/listinfo/matplotlib-users>`__
```

Comment:
Many of these redirect; it saves one request for every click for every user. Just because individually there's not much difference, doesn't mean that collectively there isn't one.

I was waiting for the gitwash PR because a whole bunch of links are fixed in there already.